src/linter.js

```diff
@@ -306,6 +306,10 @@ export default class Linter {
       const json = await this.io.getFileAsString(constants.MANIFEST_JSON);
       const manifestParser = new ManifestJSONParser(json, this.collector, {
         io: this.io,
+        // Account for unit tests where io.files isn't set.
```

Comment:
I don't understand why this is needed.

Reply:
This was something I was meant to take another look into, basically it looked like not all unit tests were mocking io.files and so some unrelated test was failing because of this additional check.

But I took another look as I was planning to, and it turns out that `io.getFiles()` is actually mocked in all there where it should and it would be a better way to retrieve the map of the assets packaged with the extensions, I changed this part and now all tests are passing without any need to relax the expectations on the implementation side. 

I'll submit that part of the chages in this PR soon.